awx/ui_next/src/components/Sort/Sort.jsx

```diff
@@ -135,7 +135,6 @@ class Sort extends React.Component {
                 onSelect={this.handleDropdownSelect}
                 direction={up}
                 isOpen={isSortDropdownOpen}
-                style={{ width: '100%', maxWidth: '100px' }}
```

Comment:
I'm happy to remove this from Search and Sort, as it doesn't work correctly.

We do, however, need to figure out a good solution for when the selected keys make the dropdown inputs too long for their space in the toolbar.  things did not look like they would responsively shrink in a nice way.

Reply:
I agree. Maybe we can return to these styles when PF moves the Toolbar from experimental?